slack_sdk/web/internal_utils.py

```diff
@@ -247,10 +247,12 @@ def _to_0_or_1_if_bool(v: Any) -> Union[Any, str]:
     return v
 
 
-def _warn_if_text_or_attachment_fallback_is_missing(endpoint: str, kwargs: Dict[str, Any]) -> None:
+def _warn_if_message_text_content_is_missing(endpoint: str, kwargs: Dict[str, Any]) -> None:
     text = kwargs.get("text")
-    if text and len(text.strip()) > 0:
+    markdown_text = kwargs.get("markdown_text")
+    if (text and len(text.strip()) > 0) or (markdown_text and len(markdown_text.strip()) > 0):
```

Comment:
suggestion(non-blocking): Separating these cases as separate `if` statements and returns might make later additions more clear 👁️‍🗨️